packages/RichTextEditing-Tests.package/RichTextEditorTestCase.class/instance/boldTextStructure.st

```diff
@@ -0,0 +1,4 @@
+constants
+boldTextStructure
```

Comment:
Isn't this already defined in TextStructure?

Reply:
convenience/ abstraction